packages/labs/virtualizer/package.json

```diff
@@ -47,7 +47,7 @@
       ]
     },
     "build:copy-polyfill": {
-      "command": "copyfiles -u 1 src/polyfills/resize-observer-polyfill/ResizeObserver.js .",
+      "command": "mkdir -p polyfills/resize-observer-polyfill && cp src/polyfills/resize-observer-polyfill/ResizeObserver.js polyfills/resize-observer-polyfill/ResizeObserver.js",
```

Comment:
This change is to remove the dep on `copyfiles`?

Reply:
Yep